test/expressions/generateExpressionTests.py

```diff
@@ -90,6 +90,24 @@ def add_extra_signatures(res):
         "vector unit_vector_constrain(row_vector, real)",
         "vector unit_vector_free(vector)",
         "vector unit_vector_free(row_vector)",
+        "vector positive_ordered_constrain(vector)",
+        "vector positive_ordered_constrain(row_vector)",
```

Comment:
I don't think any of these are actually expected to work with row_vectors (check the tests)

Reply:
idt they need to, but when I rewrote these they can now. Should I update the tests to use row vectors?

Reply:
Revert to just vectors. For instance:

```
vector positive_ordered_constrain(row_vector)
```

Doesn't really make sense (returns a `vector` give a `row_vector`). We aren't using `row_vector` here so I'd rather not bother with adding it. I wonder how that works under the hood though? Cause we can definitely positive constrain Stan arrays, etc. I'd rather not dig into that here though and get this pull in.

I'm trying to figure out how to do reverse mode on `simplex_constrain` with lp and failing :(.

Reply:
Oh yeah also I've checked all your changes up to this point again (all the reference stuff). Looks fine to me.

Reply:
Word I'll fix this then we should be good to go!